audio_service/test/data_classes_test.dart

```diff
@@ -56,22 +56,24 @@ void main() {
     });
 
     test('equality', () {
-      expect(
-        PlaybackState(
-          controls: [MediaControl.play],
-          androidCompactActionIndices: [1, 2, 3],
-          systemActions: {MediaAction.fastForward},
-        ),
-        PlaybackState(
-          controls: [MediaControl.play],
-          androidCompactActionIndices: [1, 2, 3],
-          systemActions: {MediaAction.fastForward},
-        ),
-      );
-      expect(
-        PlaybackState(controls: [MediaControl.pause]),
-        isNot(PlaybackState(controls: [MediaControl.play])),
-      );
+      fakeAsync((async) {
+        expect(
+          PlaybackState(
+            controls: [MediaControl.play],
+            androidCompactActionIndices: [1, 2, 3],
+            systemActions: {MediaAction.fastForward},
+          ),
+          PlaybackState(
+            controls: [MediaControl.play],
+            androidCompactActionIndices: [1, 2, 3],
+            systemActions: {MediaAction.fastForward},
+          ),
+        );
+        expect(
```

Comment:
This will trivially be true because it's not in `fakeAsync` and the `updateTime` will not be equal, so it probably should also be in `fakeAsync`.

Reply:
It's all inside fakeAsync, did you place the comment in the wrong place or simply overlooked it?

Reply:
Yep, my mistake. I thought the second `expect` was not inside `fakeAsync`.